### PR TITLE
Add RVM Completion

### PR DESCRIPTION
**Pull request by LanikSJ**
Adding RVM Completion from source: https://rvm.io/workflow/completion

### CODE REVIEW

File: completion/available/rvm.completion.bash [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Bash completion support for  RVM.
+# Source: https://rvm.io/workflow/completion
+
+
+_rvm_complete() {
+	[[ -r $rvm_path/scripts/completion ]] && . $rvm_path/scripts/completion
+}
+
+complete -o default -o nospace -F _rvm_complete rvm
```
Comment (nwinkler) [inline completion/available/rvm.completion.bash]: I don't think that this line is necessary. Isn't this already included in the `completion` script here? https://github.com/rvm/rvm/blob/master/scripts/completion#L23

**Comment by LanikSJ** [comment]
You're correct I shall remove it.

File: completion/available/rvm.completion.bash [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Bash completion support for  RVM.
+# Source: https://rvm.io/workflow/completion
+
+
+_rvm_complete() {
```
Comment (nwinkler) [inline completion/available/rvm.completion.bash]: I think you need to get rid of this function. From what I understand, you only really need to have the next line in this file.

File: completion/available/rvm.completion.bash [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+	[[ -r $rvm_path/scripts/completion ]] && . $rvm_path/scripts/completion
+}
```
Comment (nwinkler) [inline completion/available/rvm.completion.bash]: This line needs to go as well.

**Comment by LanikSJ** [comment]
Agreed it's kinda pointless now.  Removed.  That's what happens when I do heavy lifting at 2am. :-P

**Comment by nwinkler** [comment]
Great - thanks!

**Pull request merged**